chronoline/chronoline.js

```diff
@@ -543,6 +549,23 @@ function Chronoline(domElement, events, options) {
 
                 elem.attr('title', myEvent.title);
 
+                if (myEvent.attrs.text) {
+                    try {
+                        var tNodeVal = typeof (myEvent.attrs.text) === "string" ? myEvent.attrs.text : myEvent.attrs.text.value;
+                        var yOffset = myEvent.attrs.text.offset || 10;
+                        var tNode = t.paper.text(startX, upperY + yOffset, tNodeVal);
+                        if (myEvent.attrs.text.options) {
+                            tNode.attr(myEvent.attrs.text.options);
+                        } else if (options.eventLabels) {
```

Comment:
hmm, I believe that the logic is setup so that `options` and `defaults` are merged onto `t`, so you should just be able to do 

```
else {
   tNode.attr(t.eventLabels);
}
```

Does that work, or am I missing something here?

Reply:
This is different.  The `text` property can either be a string or an object. If it's an object, the text label comes from `value` and there is an `options` node for setting font-size, text-anchor, etc. The options that you guys have are options for the event nodes. The text label is a different Raphael element outside of this, so it could have it's own options. See below. I'll document it in a little bit. Hopefully that'll add clarity.